src/motion-queue.h

```diff
@@ -72,9 +75,27 @@ struct MotionSegment {
 #endif
 } __attribute__((packed));
 
+// Layout of the status register
+struct QueueStatus {
+  uint32_t counter : 24;
+  uint32_t index : 8;
+} __attribute__((packed));
```

Comment:
__attribute__((packed)) is probalby not needed for this one. The compiler knows what to do here.
Packed only really is needed if you have multiple fields in a struct which might trigger padding (e.g. a byte next to a int16).